### ISSUE TITLE
Support sorting upon a column

### ISSUE THREAD

**Issue by Ricky-Tigg** [flatpak/flatpak#4807]
### Checklist

- [X] I agree to follow the [Code of Conduct](https://github.com/flatpak/flatpak/blob/main/CODE_OF_CONDUCT.md) that this project adheres to.
- [X] I have searched the [issue tracker](https://www.github.com/flatpak/flatpak/issues) for a feature request that matches the one I want to file, without success.

### Suggestion

**Flatpak v. :** 1.12.6

Hello. Supporting sorting upon a chosen column would be convenient; e.g. upon _Application ID_ would allow to determine easily up-to-date and old versions
```
$ flatpak list
Name       Application ID      Version       Branch     Installation
[...]
```

**Comment by Ricky-Tigg** [comment]
Noticing lately the state of null maintenance of this project for long period. Consider closing the present report.

**Comment by mmahmoudian** [comment]
@Ricky-Tigg I was wondering why this issue was closed. Did you find a way to sort them properly without jumping through hoops in bash?

**Comment by mmahmoudian** [comment]
> Willing to assist primary maintainer? Only the reopening of the ticket by the latter would demonstrate that there is some interest in this subject.

I disagree. One way to help a project is to give them useful ideas. Reopening it shows to the maintainers that there is feature request and there are people interested in having this option. You closed the issue on the mark of 100 days (2022-03-14 .. 2022-06-24). In FLOSS world 100 days is virtually nothing.
